src/main/java/seedu/smarthomebot/logic/commands/ListCommand.java

```diff
@@ -35,20 +35,20 @@
     private static final String DISPLAY_TYPE = " | Type: ";
     private static final String DISPLAY_PARAMETER = " | Parameter: ";
 
-    private final String parameter;
+    private final String argument;
     private final String filteredLocation;
 
-    public ListCommand(String arguments, String filteredLocation) {
-        assert arguments.isEmpty() != true : "InvalidCommand must not accept empty arguments";
-        this.parameter = arguments;
+    public ListCommand(String argument, String filteredLocation) {
+        assert argument.isEmpty() != true : "InvalidCommand must not accept empty argument";
+        this.argument = argument;
         this.filteredLocation = filteredLocation;
     }
 
     @Override
     public CommandResult execute() {
         try {
             int index = 1;
-            switch (parameter) {
+            switch (argument) {
```

Comment:
same thing as the other comment: should we standardize with 'this.argument'

Reply:
As discussed, we dont need the this .